config/app.cfg

```diff
@@ -0,0 +1,24 @@
+[profiles]
+environment = dev
+db = postgresql://username:password@hostname/profiles
+server_name = profiles--dev
+scheme = http
+
+[logging]
+path = var/logs
+level = DEBUG
+
+[orcid]
+api_uri = https://api.sandbox.orcid.org
```

Comment:
all these configurations are lifted by the `.dist` files, but they will change to point to other containers: `orcid-dummy` and [`postgres`](https://hub.docker.com/_/postgres/) (official image)